source/manual/view-extract-feedback-data-bigquery.html.md.erb

```diff
@@ -0,0 +1,150 @@
+---
+owner_slack: "#govuk-data-labs"
+title: View or extract feedback data in BigQuery
+section: Data Community Documentation
+layout: manual_layout
+parent: "/manual.html"
+---
+
+You can do the following with the feedback data in BigQuery:
+
+- use Google Data Studio to present the data
+- extract the data from BigQuery into another file format, for example a CSV or a Google Sheets file
+
+You do not need a specific permission to view or extract feedback data in BigQuery once you have access to the BigQuery data sets. To access the BigQuery data, contact the GDS Data Labs team on the [#govuk-data-labs Slack channel](https://app.slack.com/client/T8GT9416G/CHR4UQKU4).
+
+BigQuery feedback data comes from SmartSurvey and Zendesk.
+
+If you want to view or extract Zendesk feedback data, you should first reformat this data.
+
+You do not need to reformat SmartSurvey data.
+
+## Reformat Zendesk data
+
+Zendesk data in BigQuery is in a different format to the SmartSurvey data, and contains the following:
+
+- feedback data
+- Zendesk tickets raised by other Government departments
+
+You should reformat the Zendesk feedback data and remove Zendesk tickets before you view or analyse that data.
+
+You do this by running a SQL query in BigQuery to:
+
+- change the data format to the same format as SmartSurvey
+- rename some data fields
+- get the URL that the feedback was left on
+- exclude Zendesk tickets raised by other Government departments by excluding unwanted URLs
+
+Once you have run this SQL query, the Zendesk feedback data will be ready to view or extract for analysis.
+
+1. Go to the [GOV.UK BigQuery project](https://console.cloud.google.com/bigquery?project=gov uk-bigquery-analytics).
+1. Select __QUERY TABLE__.
+1. Run the following SQL code in the code window:
+
+    ```sql
+    SELECT
+            cast(created_at AS TIMESTAMP) AS created_at,
+            ‘Ticket’ AS source,
```

Comment:
When I copy and paste this in I'm getting an error

`Syntax error: Illegal input character "\342" at [3:13]`, it doesn't like the single quotes around Ticket. Is that just me, or does it happen to you also?

Reply:
The same applies to all the single quotes, you'll need to replace them?